frontend/src/components/account-mgmt/ManageContactFacilityPermissions.vue

```diff
@@ -0,0 +1,250 @@
+<template>
+  <v-row>
+    <v-col class="ml-6 pr-9 pt-0 pb-0">
+      <h4>{{ title }}</h4>
+    </v-col>
+  </v-row>
+  <v-row>
+    <v-col cols="12" class="ml-6 pr-9 pt-0">
+      <v-card elevation="0" variant="outlined">
+        <v-expansion-panels variant="accordion" v-model="isPanelOpenAdditionalContacts">
+          <v-expansion-panel elevation="0">
+            <v-expansion-panel-title>
+            </v-expansion-panel-title>
+            <v-expansion-panel-text>
+              <v-skeleton-loader :loading="loading" type="table-tbody">
+                <div class="w-100">
+                  <v-row v-if="editMode">
+                    <v-col cols="auto">
+                      <AppLabel>Add {{ title }}:</AppLabel>
+                    </v-col>
+                    <v-col cols="3">
+                      <v-autocomplete
+                        id="additional-contact"
+                        ref="autoComplete"
+                        v-model="contactId"
+                        :items="sortedContactsAvailableForAdd"
+                        :disabled="loading"
+                        item-title="fullName"
+                        item-value="contactId"
+                        label="Select contact to add"
+                        density="compact"
+                        variant="outlined"
+                        @update:modelValue="addContact">
+                      </v-autocomplete>
+                    </v-col>
+                  </v-row>
+                  <v-row>
+                    <v-col class="pt-2">
+                      <v-data-table
+                        :headers="headers"
+                        :items="sortedContacts"
+                        density="compact">
+                        <template v-slot:item.name="{ item }">
+                          {{ item.firstName + ' ' + item.lastName }}
+                        </template>
+                        <template v-slot:item.role="{ item }">
+                          {{ getRoleNameById(item.role) }}
+                        </template>
+                        <template v-slot:item.actions="{ item }">
+                          <v-row v-if="editMode" justify="end">
+                            <v-icon icon="fa:fa-regular fa-trash-can" @click="deleteContact(item)"></v-icon>
+                          </v-row>
+                        </template>
+                        <template v-slot:[`header.actions`]>
+                          <v-row v-if="!editMode" justify="end">
+                            <v-icon icon="fa:fa-regular fa-edit" @click="toggleEditMode()"></v-icon>
```

Comment:
I noticed that the Edit buttons aren't disabled if you are editing another section on the page. Not sure if this will cause issues. I have some thoughts on a solution if we think it's a problem. Might be something to deal with when you work on Facility Edit.

Reply:
I thought about that and tested it... couldn't cause a break scenario although tightening it up does make sense. Sounds good to tackle it during facility update if your okay with that.

Reply:
Yeah, I think it's pretty safe but user experience could be improved. Defer to edit facility.